examples/benchmarks/cvrp_augerat.py

```diff
@@ -1,157 +1,170 @@
-        for line in df_augerat.itertuples():
-            values = line[1].split()
-            node = AugeratNodePosition(values)
-            self.G.add_node(node.name, x=node.x, y=node.y, demand=0)
-            # Add Sink as copy of Source
-            if node.name == "Source":
-                self.G.add_node("Sink", x=node.x, y=node.y, demand=0)
-
-        # Read demand from txt file
-        df_demand = read_csv(
-            path + instance_name,
-            sep="\t",
-            skiprows=range(7 + n_vertices),
-            nrows=n_vertices,
-        )
-        for line in df_demand.itertuples():
-            values = line[1].split()
-            node = AugeratNodeDemand(values)
-            self.G.nodes[node.name]["demand"] = node.demand
-
-        # Add the edges, the graph is complete
-        for u in self.G.nodes():
-            if u != "Sink":
-                for v in self.G.nodes():
-                    if v != "Source":
-                        if u != v:
-                            self.G.add_edge(u,
-                                            v,
-                                            cost=round(self.distance(u, v), 1))
-
-        # relabel
-        before = [v for v in self.G.nodes() if v not in ["Source", "Sink"]]
-        after = [v - 1 for v in self.G.nodes() if v not in ["Source", "Sink"]]
-        mapping = dict(zip(before, after))
-        self.G = relabel_nodes(self.G, mapping)
-
-    def distance(self, u, v):
-        """2D Euclidian distance between two nodes.
-
-        Args:
-            u (Node) : tail node.
-            v (Node) : head node.
-
-        Returns:
-            float : Euclidian distance between u and v
-        """
-        delta_x = self.G.nodes[u]["x"] - self.G.nodes[v]["x"]
-        delta_y = self.G.nodes[u]["y"] - self.G.nodes[v]["y"]
-        return round(sqrt(delta_x**2 + delta_y**2), 0)
-
-    def solve(self,
-              initial_routes=None,
-              cspy=False,
-              num_stops=None,
-              exact=True,
-              time_limit=None,
-              pricing_strategy="BestPaths",
-              dive=False,
-              greedy=False):
-        """Instantiates instance as VRP and solves."""
-        if cspy:
-            self.G.graph["subproblem"] = "cspy"
-        else:
-            self.G.graph["subproblem"] = "lp"
-        print(self.G.graph["name"], self.G.graph["subproblem"])
-        print("===========")
-        prob = VehicleRoutingProblem(
-            self.G,
-            load_capacity=self.max_load,
-            num_stops=num_stops,
-        )
-        prob.solve(initial_routes=initial_routes,
-                   cspy=cspy,
-                   exact=exact,
-                   time_limit=time_limit,
-                   pricing_strategy=pricing_strategy,
-                   dive=dive,
-                   greedy=greedy)
-        self.best_value, self.best_routes = prob.best_value, prob.best_routes
+from math import sqrt

```

Comment:
Don't understand why there's this much diff, for no changes...

Reply:
Please try merging the original vrpy dev branch into yours (can be done using a pull request) and solving the merge conflicts.